src/views/CountryRiskWatch/RiskBarChart/WildfireChart/index.tsx

```diff
@@ -0,0 +1,204 @@
+import { paths } from '#generated/riskTypes';
+import { isDefined, listToGroupList, mapToList } from '@togglecorp/fujs';
+
+import { avgSafe, formatNumber } from '#utils/common';
+import { getDiscretePathDataList, getPathData, getScaleFunction } from '#utils/chart';
+import { useMemo, useRef } from 'react';
+import useSizeTracking from '#hooks/useSizeTracking';
+import {
+    COLOR_LIGHT_GREY,
+    COLOR_PRIMARY_BLUE,
+    COLOR_PRIMARY_RED,
+} from '#utils/constants';
+
+import ChartAxes from '../ChartAxes';
+import styles from './styles.module.css';
+
+type GetCountryRisk = paths['/api/v1/country-seasonal/']['get'];
+type CountryRiskResponse = GetCountryRisk['responses']['200']['content']['application/json'];
+type RiskData = CountryRiskResponse[number];
+
+const X_AXIS_HEIGHT = 24;
+const Y_AXIS_WIDTH = 32;
+const CHART_OFFSET = 16;
+
+const chartMargin = {
+    left: Y_AXIS_WIDTH + CHART_OFFSET,
+    top: CHART_OFFSET,
+    right: CHART_OFFSET,
+    bottom: X_AXIS_HEIGHT + CHART_OFFSET,
+};
+
+const formatDate = (date: Date) => date.toLocaleString(
+    navigator.language,
+    { month: 'short' },
+);
+
+interface ChartPoint {
+    x: number;
+    y: number;
+    label: string | undefined;
+}
+
+function chartPointSelector(chartPoint: ChartPoint) {
+    return chartPoint;
+}
+
+interface Props {
+    gwisData: RiskData['gwis'] | undefined;
+}
+
+function WildfireChart(props: Props) {
+    const { gwisData } = props;
+
+    const chartContainerRef = useRef<HTMLDivElement>(null);
+    const chartBounds = useSizeTracking(chartContainerRef);
+
+    const points = useMemo(
+        () => {
+            const monthGroupedData = listToGroupList(
+                gwisData?.filter((dataItem) => dataItem.dsr_type === 'monthly') ?? [],
+                (gwisItem) => gwisItem.month,
+            );
+
+            const currentYear = new Date().getFullYear();
+
+            const aggregatedList = mapToList(
+                monthGroupedData,
+                (monthlyData, monthKey) => {
+                    const average = avgSafe(monthlyData.map((dataItem) => dataItem.dsr)) ?? 0;
+                    const min = avgSafe(monthlyData.map((dataItem) => dataItem.dsr_min)) ?? 0;
+                    const max = avgSafe(monthlyData.map((dataItem) => dataItem.dsr_max)) ?? 0;
```

Comment:
So this is the average of average, min and max?

Reply:
Yes